### PR TITLE
Add Procedure for Disabling DAG-Only Deploys

### LINKED ISSUE #1627: Add docs on disabling DAG only Deploys
### Context

When we initially released DAG deploys, we decided to not document how to disable the feature. However, it has seen become more obvious that we're just unnecessarily obfuscating an easy action for users that they can also naturally infer from `astro deployment update --dag-deploy enable`

### Ask

- [ ] Add new section on disabling dag deploys
- [ ] Add flag to CLI command reference

Make sure to include a WARNING about what happens when you disable:

1. Run the command
2. Run `astro deploy` right after

Check the warning copy in the Astro CLI for details -- the behavior is akin to DAGs in Airflow UI disappearing until you push code again. cc: @sunkickr @kushalmalani @wolfier

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1627

### CODE REVIEW

**Comment by kwcanuck** [comment]
@jwitz ready for peer review. It should be quick.

**Review by jwitz — APPROVED** [review]
LGTM, do we need to warn users about taking this action @paolaperaza ?

File: astro/deploy-code.md [hunk 1]
```diff
@@ -150,6 +150,20 @@ Run the following command to deploy only your `dags` directory to a Deployment:
 astro deploy --dags
 ```
 
+### Disable DAG-only deploys on a Deployment
+
```
Comment (paolaperaza) [inline astro/deploy-code.md:154]: Let's add some context to this action:

- A statement on why a user would want to disable DAG only deploys and that they can do it any time
- What actually happens to your DAGs and Deployment when you do it
- Consider specifying that only a Workspace Admin can do this
- Consider recommending that the user reach out to CRE prior to taking this action

@sunkickr Can you help confirm what happens when you disable DAG deploys?

File: astro/deploy-code.md [hunk 2]
```diff
@@ -156,6 +156,20 @@ Run the following command to deploy only your `dags` directory to a Deployment:
 astro deploy --dags
 ```
 
+### Disable DAG-only deploys on a Deployment
```
Comment (kwcanuck) [inline astro/deploy-code.md:159]:
```suggestion
### Turn off DAG-only deploys on a Deployment
```

File: astro/deploy-code.md [hunk 3]
```diff
@@ -156,6 +156,20 @@ Run the following command to deploy only your `dags` directory to a Deployment:
...
+
+1. Run the following command to disable DAG-only deploys:
```
Comment (kwcanuck) [inline astro/deploy-code.md:161]:
```suggestion
1. Run the following command to turn off DAG-only deploys:
```

**Comment by kwcanuck** [comment]
@paolaperaza and @jwitz ready for your review.

File: astro/deploy-code.md [hunk 4]
```diff
@@ -156,6 +156,23 @@ Run the following command to deploy only your `dags` directory to a Deployment:
 astro deploy --dags
 ```
 
+### Turn off DAG-only deploys on a Deployment
```
Comment (paolaperaza) [inline astro/deploy-code.md:159]:
```suggestion
### Disable DAG-only deploys on a Deployment
```

- Why not the word `disable`, to be consistent with `enable` and the CLI command itself? I think we can keep this H3, and then keep `turn off` in the first sentence to be more explicitly clarifying language

File: astro/deploy-code.md [hunk 5]
```diff
@@ -156,6 +156,23 @@ Run the following command to deploy only your `dags` directory to a Deployment:
...
+
+You can turn off DAG-only deploys any time the functionality is no longer required. When you turn off DAG-only deploys, new DAGs are not pushed to your Deployments and all existing DAGs are removed from your Deployments. Astronomer recommends running an image deploy immediately after you turn off DAG-only deploy functionality to make sure all of your DAG files are available to your Deployments. You can contact [Astronomer support](https://cloud.astronomer.io/support) to determine if turning off DAG-only deploy functionality is the right choice for your organization. 
+
+You need Workspace Admin permissions to turn off DAG-only deploys.
```
Comment (paolaperaza) [inline astro/deploy-code.md:163]:
```suggestion
```

File: astro/deploy-code.md [hunk 6]
```diff
@@ -156,6 +156,23 @@ Run the following command to deploy only your `dags` directory to a Deployment:
...
+You can turn off DAG-only deploys any time the functionality is no longer required. When you turn off DAG-only deploys, new DAGs are not pushed to your Deployments and all existing DAGs are removed from your Deployments. Astronomer recommends running an image deploy immediately after you turn off DAG-only deploy functionality to make sure all of your DAG files are available to your Deployments. You can contact [Astronomer support](https://cloud.astronomer.io/support) to determine if turning off DAG-only deploy functionality is the right choice for your organization. 
+
+You need Workspace Admin permissions to turn off DAG-only deploys. 
+1. Run the following command to turn off DAG-only deploys:
+
+    ```sh
+    astro deployment update --dag-deploy disable
+    ```
+
+2. Run the following command to push your changes to Astro:
```
Comment (paolaperaza) [inline astro/deploy-code.md:170]:
```suggestion
2. Run the following command to deploy all files in your Astro project as a Docker image:
```

**Review by paolaperaza — COMMENTED** [review]
Couple comments here @kwcanuck -- LMK what you think

File: astro/deploy-code.md (hunk 4 above)
Comment (kwcanuck) [reply astro/deploy-code.md:159]: From the Google developer documentation style guide:

image

Like _blacklist_ or _slave_, _disable_ is not an inclusive term.

**Pull request merged**